src/python/pants/backend/jvm/tasks/jvm_compile/javac/javac_compile.py

```diff
@@ -245,7 +245,7 @@ def _execute_hermetic_compile(self, cmd, ctx):
             if f.endswith(".java")
         )
 
-        exec_process_request = ExecuteProcessRequest(
+        exec_process_request = Process(
```

Comment:
Consider renaming these variables to either `process` or `process_request`. Ditto on all the other call sites in this PR. (Sorry, I know this is tedious)